README.md

```diff
@@ -49,6 +51,18 @@ Make sure Docker can be executed without sudo for the scripts in this guide to w
     
 Log out and login again; or run `sudo su - $USER` command to continue.
 
+#### Podman on macOS
+
+Install Podman Desktop:
+
+    brew install podman-desktop
+
+Create a new Podman Machine with sufficient resources (Preferences - Resources - Create new ...)
+* CPUs: Minimum 2. Use 8 or more for faster build times.
+* Memory: Minimum 8 GB. 16 GB or more is preferred.
+* Disk size: 128 GB.
+* Machine with root privileges: Enabled
```

Comment:
Should we have a [`podman machine init`](https://docs.podman.io/en/stable/markdown/podman-machine-init.1.html) one-liner here with suggested parameters?

Something like
```
podman machine init --cpus=8 --disk-size=128 --memory=16384 --rootful
```
Note: not manually tested 👀